package.json

```diff
@@ -18,10 +18,12 @@
   "files": [
     "index.js"
   ],
+  "types": "./index.d.ts",
   "dependencies": {
     "is-empty": "^1.0.0"
   },
   "devDependencies": {
```

Comment:
Could `dtslint` be added as a dev dependency, and `dtslint types` be added as a `test-types` script?